tests/zfs-tests/tests/functional/cli_root/zhack_scrub/zhack_scrub_001_pos.ksh

```diff
@@ -0,0 +1,52 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2007 Sun Microsystems, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+
+#
+# Copyright (c) 2016 by Delphix. All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/cli_root/zhack_scrub/zhack_scrub.cfg
+
+# DESCRIPTION:
+#	Verify that we can find an exported pool and scrub it
+#
+# STRATEGY:
+#	1. Create pool and fill with hundreds data.
+#	2. zpool export it
+#	3. zhack scrub the pool
+
+verify_runnable "global"
+
+log_assert "Verify zhack scrub works"
+log_must zinject -d $DISK1 -D10:1 $TESTPOOL
```

Comment:
In master we recently added some test infrastructure to make this check easier, take a look at cef48f14da6eec8a9095a0272f1cd7d9d60ca476.

Reply:
Hm.  I think in fact the `zinject` commands are irrelevant, copied from whatever test suite I stole the example from.  I'm not trying to race `zinject` and `zhack scrub` here; I've dropped the lines.